components/provider/src/resource.rs

```diff
@@ -104,8 +106,8 @@ macro_rules! resource_key {
     (plurals, $sub_category:literal, $version:tt) => {
         $crate::resource_key!($crate::ResourceCategory::Plurals, $sub_category, $version)
     };
-    (dates, $sub_category:literal, $version:tt) => {
-        $crate::resource_key!($crate::ResourceCategory::Dates, $sub_category, $version)
+    (timezones, $sub_category:literal, $version:tt) => {
+        $crate::resource_key!($crate::ResourceCategory::TimeZones, $sub_category, $version)
```

Comment:
This change may look confusing, but this was actually a duplicate branch in the macro.

If you look at the first macro branch, you'll see it is the same as what this one used to be. 

So this is effectively adding a new branch (even though the change looks like I'm replacing an existing branch).